chart/slackconnector/templates/application.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: applicationconnector.kyma-project.io/v1alpha1
+kind: Application
+metadata:
+  name: {{ .Release.Name }}-app
+  labels:
+    app: {{ include "slackconnector.name" . }}
+    chart: {{ include "slackconnector.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    app.kubernetes.io/name: {{ template "slackconnector.name" . }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
```

Comment:
Labels `chart`, `release` and `heritage` are basically an older version of `helm.sh/chart`, `app.kubernetes.io/managed-by`, `app.kubernetes.io/instance`. I do not think you need all of them. 
I would suggest sticking to the standard ones https://helm.sh/docs/chart_best_practices/#standard-labels and remove `chart`, `release`, `heritage`.